modules/tinymce/src/themes/silver/main/ts/ui/dialog/Label.ts

```diff
@@ -9,10 +9,11 @@ import { RepresentingConfigs } from '../alien/RepresentingConfigs';
 type LabelSpec = Omit<Dialog.Label, 'type'>;
 
 export const renderLabel = (spec: LabelSpec, backstageShared: UiFactoryBackstageShared): SimpleSpec => {
+  const endClass = spec.align === 'end' ? [ 'tox-label--end' ] : [];
   const label: AlloySpec = {
     dom: {
       tag: 'label',
-      classes: [ 'tox-label' ]
+      classes: [ 'tox-label', ...endClass ]
```

Comment:
```suggestion
  const classes = [ 'tox-label',  ...(spec.align === 'end' ? [ 'tox-label--end' ] : []) ];
  const label: AlloySpec = {
    dom: {
      tag: 'label',
      classes
```

Reply:
After adding center alignment, it became cleaner to not inline the conditional classes.